<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-4G6V73A" failures="4" tests="5" name="Register Page Sucess Test Cases chrome" time="40.7" errors="0" timestamp="2022-11-29T09:51:01 IST">
  <testcase classname="TestLayer.RegisterPageTest" name="validateRegisterFunctionality" time="11.378"/>
  <testcase classname="TestLayer.RegisterSucessTest" name="validateSucessPageUrl" time="0.268">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at TestLayer.RegisterSucessTest.validateSucessPageUrl(RegisterSucessTest.java:17)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- validateSucessPageUrl -->
  <testcase classname="TestLayer.RegisterSucessTest" name="validateFirstName" time="0.301">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at TestLayer.RegisterSucessTest.validateFirstName(RegisterSucessTest.java:22)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- validateFirstName -->
  <testcase classname="TestLayer.RegisterSucessTest" name="validateLastName" time="0.243">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at TestLayer.RegisterSucessTest.validateLastName(RegisterSucessTest.java:27)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- validateLastName -->
  <testcase classname="TestLayer.RegisterSucessTest" name="validateEmailId" time="0.077">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at TestLayer.RegisterSucessTest.validateEmailId(RegisterSucessTest.java:32)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- validateEmailId -->
</testsuite> <!-- Register Page Sucess Test Cases chrome -->
